framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ServiceProxying/Angular/AngularServiceProxyGenerator.cs

```diff
@@ -48,7 +48,7 @@ public override async Task GenerateProxyAsync(GenerateProxyArgs args)
         var apiName = args.ApiName ?? defaultValue;
         var source = args.Source ?? defaultValue;
         var target = args.Target ?? defaultValue;
-
+        var url = args.Url ?? "''";
```

Comment:
How about using `string.Empty` here? Is there any special case for using empty char here?

```suggestion
        var url = args.Url ?? string.Empty;
```